examples/utils/bundle-size-tests/src/aqueduct.ts

```diff
@@ -9,17 +9,26 @@ import {
 } from "@fluidframework/aqueduct";
 import { DirectoryFactory } from "@fluidframework/map";
 import { SharedStringFactory } from "@fluidframework/sequence";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
 
 export function apisToBundle() {
 	class BundleTestDo extends DataObject {}
-	const doFactory = new DataObjectFactory(
+	const defaultFactory = new DataObjectFactory(
 		"BundleTestDo",
 		BundleTestDo,
 		[new SharedStringFactory(), new DirectoryFactory()],
 		{},
 	);
 
-	new ContainerRuntimeFactoryWithDefaultDataStore(doFactory, [
-		["BundleTestDo", Promise.resolve(doFactory)],
-	]);
+	new ContainerRuntimeFactoryWithDefaultDataStore({
+		defaultFactory,
+		registryEntries: [["BundleTestDo", Promise.resolve(defaultFactory)]],
+		initializeEntryPoint: async (runtime: IContainerRuntime) => {
+			const entryPoint = await runtime.getAliasedDataStoreEntryPoint("default");
+			if (entryPoint === undefined) {
+				throw new Error("default dataStore must exist");
+			}
+			return entryPoint.get();
```

Comment:
Nit: maybe `entryPoint -> dataStoreHandle`? It feels a bit weird that in a function which is supposed to initialize the entryPoint, there's a variable called `entryPoint` but it's not directly what we return.